component-test/service_stubs.py

```diff
@@ -103,19 +127,23 @@ def __init__(self, bus):
 
         # Some nonsensical properties so there is something to return
         # from GetAll
-        self.__properties = {"MyKey1": "my_value_1", "MyKey2": "my_value_2"}
+        self.__properties = dict()
+        ifaces = [TestInterface1_1, TestInterface1_1_2]
+        for iface in ifaces:
+            self.__properties[iface] = {PROP_KEY_1 + iface: PROP_VALUE_1 + iface,
+                                        PROP_KEY_2 + iface: PROP_VALUE_2 + iface}
```

Comment:
Definitely not, it was mostly a comment on the discussion we had over the last few weeks on the topic of hardcoded variables etc ;). I digress.